docs/rules/no-negated-condition.md

```diff
@@ -56,8 +34,7 @@ if (a !== b) {
 
 ```
 
-The following patterns are not warnings:
-
+Examples of **correct** code for this rule:
```

Comment:
It seems like the correct code could be more carefully inverted compared to the incorrect code. For example:

```
a ? c : b;
```